### PR TITLE
Adding drop decimals support

### PR DESCRIPTION
**Pull request by cjdodson**
Added functionality to the submodule to drop decimals in the Possible Time Save component, with checkbox in the settings to enable/disable. (https://github.com/LiveSplit/LiveSplit/pull/2194)
Screenshot (195)
Screenshot (194)

### CODE REVIEW

File: UI/Components/PossibleTimeSave.cs [hunk 1]
```diff
@@ -32,6 +32,7 @@ public PossibleTimeSave(LiveSplitState state)
                 CurrentState = state
             };
             state.ComparisonRenamed += state_ComparisonRenamed;
+
```
Comment (wooferzfg) [inline UI/Components/PossibleTimeSave.cs]: We don't need this newline here.

File: UI/Components/PossibleTimeSaveSettings.cs [hunk 2]
```diff
@@ -111,6 +115,11 @@ void cmbGradientType_SelectedIndexChanged(object sender, EventArgs e)
             GradientString = cmbGradientType.SelectedItem.ToString();
         }
 
+        void chkDropDecimals_CheckedChanged(object sender, EventArgs e)
+        {
+            DropDecimals = !DropDecimals;
+        }
```
Comment (wooferzfg) [inline UI/Components/PossibleTimeSaveSettings.cs]: Are you sure that we need this here? The data bindings might handle this automatically, but please test that to make sure.

File: UI/Components/PossibleTimeSaveSettings.cs [hunk 3]
```diff
@@ -179,5 +190,10 @@ private void ColorButtonClick(object sender, EventArgs e)
         {
             SettingsHelper.ColorButtonClick((Button)sender, this);
         }
+
+        private void groupBox4_Enter(object sender, EventArgs e)
+        {
+
+        }
```
Comment (wooferzfg) [inline UI/Components/PossibleTimeSaveSettings.cs]: This isn't necessary.

File: UI/Components/PossibleTimeSaveSettings.cs (hunk 2 above)
Comment (cjdodson) [reply UI/Components/PossibleTimeSaveSettings.cs]: Oh, I guess we don't need it. I added it to make it more like the seconds/tenths/hundredths radio buttons, but I didn't realize those didn't have data bindings. (Tested to make sure)

File: UI/Components/PossibleTimeSaveSettings.cs [hunk 4]
```diff
@@ -89,6 +92,7 @@ void PossibleTimeSaveSettings_Load(object sender, EventArgs e)
             rdoSeconds.Checked = Accuracy == TimeAccuracy.Seconds;
             rdoTenths.Checked = Accuracy == TimeAccuracy.Tenths;
             rdoHundredths.Checked = Accuracy == TimeAccuracy.Hundredths;
+            chkDropDecimals.Checked = DropDecimals;
```
Comment (wooferzfg) [inline UI/Components/PossibleTimeSaveSettings.cs]: I don't think this is necessary either, but please test and verify.

Comment (cjdodson) [reply UI/Components/PossibleTimeSaveSettings.cs]: Verified unnecessary, removed in next commit

**Pull request merged**